easydock/database.py

```diff
@@ -310,6 +310,14 @@ def init_db(db_fname: str, input_fname: str, ncpu: int, max_stereoisomers=1, pre
         cur.executemany(f'INSERT INTO mols (id, stereo_id, smi, source_mol_block_input, source_mol_block) VALUES(?, ?, ?, ?, ?)', data_mol)
         conn.commit()
 
+def check_init_status_db(db_fname: str, db_col_list: str):
```

Comment:
It seems this function will be used in other cases as well, therefore I suggest to rename it to `check_db_status`